### PR TITLE
Computed metrics

### PR DESCRIPTION
**Pull request by Grumblesmith**
Adding computed metrics to this package per documented configuration.

### CODE REVIEW

**Comment by dgeorg42** [comment]
@Createor - I suspect that this is my fault, but the documentation is incorrect. Those computed metrics should **not** be created. I had documented them on Confluence because I was intending to implement them, but then realized that they were wrong. I must have forgotten to delete them from the Confluence page, and hence they ended up in the online docs. (As a side note, the Confluence pages are the source of truth for what should be implemented. If you find a difference between what's in Confluence and what's in the online docs, odds are good that Confluence was changed and the online docs were not updated.)

OK, so why are these computed metrics wrong? 

Let's start with the error rate for the GetRecords call. The collected metrics that we have to work with are GetRecords.Records and GetRecords.Success. The problem is that they are measuring two different things. GetRecords.Records is the total number of _records_ returned by calls to the GetRecords function, while GetRecords.Success is the total number of successful _calls_ to the function. So, if GetRecords.Records is 100 and GetRecords.Success is 5, all we know is that there were 5 successful calls and the total number of records returned by those calls was 100 - we have no idea how many failed calls there were (if any) or how many total calls were made. Hence, we can't compute an error rate.

Moving on to PutRecords, it's even more complicated. For one thing, there are actually _two_ functions to account for. One is PutRecord (which takes only 1 record), and the other is PutRecords (which takes multiple records).  For PutRecord (1), we have PutRecord.Success but we have no metric for the total number of calls made, or for the number of failures. PutRecords (multiple) suffers from the same problem as GetRecords, except that it's even worse because PutRecords.Success (the number of successful calls) will be incremented as long as at least one of the records in a call was put successfully. So if PutRecords.Records is 20 and PutRecords.Success is 2, then all we really know is that 2 calls were at least partially successful, but we have no idea how many total calls were made, how many calls failed completely (if any), or how many records failed (if any).  So, again, we can't compute any error rates.

Long story short - remove these computed metrics, delete them from the Confluence page, and delete them from the online docs. :)

**Comment by dgeorg42** [comment]
@Createor - I would strongly recommend against renaming the JSON files.  Those files are created and named automatically by the package export process. Presumably, you added the computed metrics by modifying the package files directly, as opposed to making the changes in the Research account in UAT and then exporting the package from there. I would advise to do it the latter way, as it allows you to test all of the changes in the Research account before exporting the packages.  Anyway, it's not my call, but figured I'd throw my 2 cents in. :)

**Comment by Grumblesmith** [comment]
Hi @dgeorg42,
Thanks for the clarification. I'll go ahead and make sure we remove those metrics from the documentation
